dsl/camel-jbang/camel-jbang-core/src/test/java/org/apache/camel/dsl/jbang/core/commands/DependencyRuntimeTest.java

```diff
@@ -0,0 +1,96 @@
+package org.apache.camel.dsl.jbang.core.commands;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class DependencyRuntimeTest extends CamelCommandBaseTest {
+
+    @TempDir
+    File tempDir;
+
+    DependencyRuntime command;
+    Path pomFilePath;
+
+    private static Stream<Arguments> testOutputArguments() {
+        return Stream.of(Arguments.of(TestArguments.QUARKUS_POM, TestArguments.QUARKUS_POM_OUTPUT));
```

Comment:
I still need to add more demo pom.xml files, but I wanted to gather feedback in the meantime. :pray: